src/openrct2/core/Stopwatch.hpp

```diff
@@ -1,52 +0,0 @@
-#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```

Comment:
I had a conflict with the file, I checked the original source and it doesn't exist anymore.